src/ButtonInteractions/reactionRoleContentEdit.ts

```diff
@@ -0,0 +1,79 @@
+import type {
+    ButtonInteraction,
+    ModalActionRowComponentBuilder,
+    TextBasedChannel,
+} from 'discord.js';
+import {
+    ActionRowBuilder,
+    EmbedBuilder,
+    ModalBuilder,
+    TextInputBuilder,
+    TextInputStyle,
+} from 'discord.js';
+import ms from 'ms';
+import ReactionRole from '../base/ReactionRole';
+
+export default class ReactionDataAdd extends ReactionRole {
+    constructor() {
+        super();
+    }
+    async execute(button: ButtonInteraction) {
+        await this.getReactionRole(button);
+        if (!this.reactionRole) return;
+        const oldMessage = await (
+            (await button.guild?.channels.fetch(
+                this.reactionRole.channelId
+            )) as TextBasedChannel
+        )?.messages.fetch(this.reactionRole.messageId);
+        const modal = new ModalBuilder()
+            .setCustomId('contentModal')
+            .setTitle('Edit the embed message content');
+        const modalTitleField = new TextInputBuilder()
+            .setCustomId('titleField')
+            .setLabel('Title')
+            .setStyle(TextInputStyle.Short)
+            .setRequired(true)
+            .setPlaceholder(oldMessage?.embeds[0]?.title || 'Reaction Role')
+            .setValue(oldMessage?.embeds[0]?.title || 'Reaction Role')
+            .setMaxLength(256);
+        const modalContentField = new TextInputBuilder()
+            .setCustomId('contentField')
+            .setLabel('Content')
+            .setStyle(TextInputStyle.Paragraph)
+            .setRequired(true)
+            .setPlaceholder(
+                oldMessage.embeds[0]?.description ||
```

Comment:
this will result in the following error if the previous description is more than 100 characters long:
```
ExpectedConstraintError > s.string.lengthLessThanOrEqual
Invalid string length
Expected: expected.length <= 100
```

How about using generous Placeholders like `Embed content`? We could also use `Modal title` as Placeholder for `modalTitleField`